django/contrib/admin/static/admin/js/calendar.js

```diff
@@ -1,3 +1,4 @@
+/*global gettext, get_format, quickElement, removeChildren*/
```

Comment:
It tells the linter what names this file expects to already be defined in the global namespace (that is, on the `window` object) by other JS files loaded before this one.